docs/application/native/tutorials/details/app-filtering.md

```diff
@@ -438,6 +443,11 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
 | `http://tizen.org/feature/sensor.tiltmeter.wakeup` | Specify this key, if the application requires the tilt sensor wake-up feature. | 2.2.1 |
 | `http://tizen.org/feature/sensor.ultraviolet` | Specify this key, if the application requires a ultraviolet sensor. | 2.3   |
 | `http://tizen.org/feature/sensor.wrist_up` | Specify this key, if the application requires a wrist up sensor. | 2.3   |
+| `http://tizen.org/feature/systemsetting.home_screen` | Specify this key, if the application requires a home screen. | 4.0   |
+| `http://tizen.org/feature/systemsetting.lock_screen` | Specify this key, if the application requires a home screen. | 4.0   |
```

Comment:
Shouldn't it be lock screen?
Specify this key, if the application requires a lock screen.

Reply:
I fixed it